src/Esfa.Vacancy.Register.Infrastructure/Repositories/FrameworkCodeRepository.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Threading.Tasks;
+using Dapper;
+using Esfa.Vacancy.Register.Domain.Repositories;
+using Esfa.Vacancy.Register.Infrastructure.Settings;
+
+namespace Esfa.Vacancy.Register.Infrastructure.Repositories
+{
+    public class FrameworkCodeRepository : IFrameworkCodeRepository
+    {
+        private readonly IProvideSettings _settings;
+
+        private const string GetFrameworkCodesQuery = @"
+            SELECT 
```

Comment:
Let's start putting this in the DB.

Reply:
We can create a follow up task for moving all the sqls into db